console/src/main/scala/io/joern/console/cpgcreation/package.scala

```diff
@@ -15,9 +15,11 @@ package object cpgcreation {
                               config: FrontendConfig,
                               rootPath: Path,
                               args: List[String]): Option[CpgGenerator] = {
+    lazy val cCpgGenerator = CCpgGenerator(config.withArgs(args), rootPath)
+    lazy val fuzzycCpgGenerator = FuzzyCCpgGenerator(config.withArgs(args), rootPath)
     language match {
       case Languages.CSHARP          => Some(CSharpCpgGenerator(config.withArgs(args), rootPath))
-      case Languages.C               => Some(FuzzyCCpgGenerator(config.withArgs(args), rootPath))
+      case Languages.C               => Seq(fuzzycCpgGenerator, cCpgGenerator).filter(_.isAvailable).headOption
```

Comment:
Could also be `find` which will stop after finding the first matching element.

Reply:
👍🏻 not so much for performance, but i like the conciseness. 
done